### ISSUE TITLE
Linking to expanded schemas

### ISSUE THREAD

**Issue by alexibz** [Redocly/redoc#1393]
Is it possible to link to an expanded schema, where you could define which properties are expanded and which option is selected in a dropdown?

**Comment by adamaltman** [comment]
No, it’s not. But this is something we’ve discussed. It’s just that in terms of time and priority we want to try to focus towards the general 2.0 release. This would probably be post-2.0.
